data_dig.sql

```diff
@@ -0,0 +1,147 @@
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (85, 'Bald', 'Male', 'Karuri', '2/28/1995', 15, 359);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (86, 'Nelli', 'Female', 'Egvekinot', '1/11/2000', 80, 109);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (87, 'Homerus', 'Male', 'Mỹ Thọ', '7/1/1992', 3, 142);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (88, 'Janean', 'Female', 'Ängelholm', '12/23/1991', 35, 275);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (89, 'Upton', 'Male', 'Palmerston North', '10/25/1991', 24, 410);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (90, 'Lindi', 'Female', 'Rancabugel', '6/24/1991', 77, 423);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (91, 'Laura', 'Female', 'Moycullen', '4/11/1997', 90, 469);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (92, 'Marcella', 'Female', 'Bataiporã', '6/6/1992', 82, 254);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (93, 'Ailey', 'Female', 'Hexiangqiao', '1/30/1998', 55, 90);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (94, 'Ava', 'Female', 'São João da Barra', '10/10/1992', 100, 41);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (95, 'Daron', 'Male', 'Chunjiang', '1/2/1997', 99, 82);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (96, 'Tallou', 'Female', 'Capellanía', '2/20/1994', 84, 175);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (97, 'Coraline', 'Female', 'São Leopoldo', '10/21/1991', 73, 171);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (98, 'Mirelle', 'Female', 'Malaruhatan', '7/29/1999', 69, 249);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (99, 'Jacquelyn', 'Female', 'San Jose', '5/2/1994', 67, 163);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (100, 'Myrna', 'Female', '''s-Hertogenbosch', '3/2/1994', 63, 156);
+
+SELECT MAX(cash)
+  FROM mock_user_data;
+
+SELECT MIN(cash)
+  FROM mock_user_data;
+
+SELECT ROUND(AVG(cash))
+  FROM mock_user_data;
+
+SELECT first_name, lucky_number AS lucky_total
+	FROM mock_user_data
+ GROUP BY first_name
+HAVING first_name LIKE "K%";
+
+SELECT COUNT(*),
+	 CASE
```

Comment:
CASE can be on the line above. WHEN can be indented under SELECT similar to what you have on line 114 and 115.